src/types/parser.ts

```diff
@@ -28,6 +28,7 @@ import {IdentifierT} from './identifiers';
 
 
 type ParsedBaseEntity = {
+	// TODO: Stop using this field to store the type of the parsed entity as Series uses this for the type of its items!
```

Comment:
That is very unfortunate indeed.
I had not realized there would be a collision when reviewing the Series project

Reply:
Now dropped, see my other comment.